pkg/common/scheduler/statefulset/scheduler.go

```diff
@@ -128,8 +129,8 @@ func (s *StatefulSetScheduler) Schedule(vpod scheduler.VPod) ([]duckv1alpha1.Pla
 		return placements, err
 	}
 	sort.SliceStable(placements, func(i int, j int) bool {
-		podIOrdinal, _ := strconv.Atoi(placements[i].PodName[len(placements[i].PodName)-1:])
-		podJOrdinal, _ := strconv.Atoi(placements[j].PodName[len(placements[j].PodName)-1:])
+		podIOrdinal, _ := strconv.Atoi(placements[i].PodName[strings.Index(placements[i].PodName, "adapter-")+8:])
+		podJOrdinal, _ := strconv.Atoi(placements[j].PodName[strings.Index(placements[j].PodName, "adapter-")+8:])
```

Comment:
can you use this helper function: https://github.com/knative-sandbox/eventing-kafka/blob/main/pkg/common/scheduler/statefulset/helpers.go#L28